app/src/modules/users/routes/item.vue

```diff
@@ -289,21 +289,12 @@ export default defineComponent({
 			usePermissions(ref('directus_users'), item, isNew);
 
 		// These fields will be shown in the sidebar instead
-		const fieldsDenyList = [
-			'id',
-			'external_id',
```

Comment:
`external_id` seems to be added back in alpha stage via https://github.com/directus/directus/commit/7be23f73d546cde54a47ad57d71de78ac794d85a#diff-74263ca7f7a8d615d67fd5fc57527f9e5ed0da2335794d1041ede28a7648f2a6R196, and I can't seem to find it being used anywhere, so I've removed it alongside this PR. Mentioning this just in case this can cause any unexpected changes that I'm not aware of.